pandas/core/frame.py

```diff
@@ -4798,9 +4800,10 @@ def dropna(self, axis=0, how="any", thresh=None, subset=None, inplace=False):
 
     def drop_duplicates(self, subset=None, keep="first", inplace=False):
         """
-        Return DataFrame with duplicate rows removed, optionally only
-        considering certain columns. Indexes, including time indexes
-        are ignored.
+        Return DataFrame with duplicate rows removed.
+
+        Optionally only considers certain columns.
+        Indexes, including time indexes are ignored.
```

Comment:
```suggestion
        Indexes, including time indexes, are ignored.
```

Reply:
Hello @gfyoung !
Added the punctuation as suggested.